src/rpc-interface.js

```diff
@@ -1,4 +1,5 @@
 "use strict";
+var ee = require("event-emitter");
```

Comment:
Same comment as above, prefer to avoid obscure variable names like `ee` if possible.